packages/btfs-http-client/protos/gogo.proto

```diff
@@ -0,0 +1,141 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
can we move all protos related files into go-btfs-common and use submodule or another type of javascript-inspired referencing method? that way to reduce the proto update conflicts in the future as they can be generated in the same project as the go pb files

Reply:
"A standard way is to create a js-btfs-common repo and publish (to npm), then import in js-btfs as an dependency so that they can totally be decoupling." (Jackie Shen). @taiyangc Would this be okay ?

Reply:
is it possible to package js-btfs-common as part of go-btfs-common? not a separate package? the reason is that we shouldn't sync *.proto files in multiple places - will be a mess later on

Reply:
Jackie and I discussed several ways of "packaging". Both Francesco and Jackie worked on packaging *A* as apart of *B* (considering A is a subdirectory of B), and found it quite difficult to pull off. In other words, sub-moduling just a subdirectory (where the js files are) is over his head. He believes it might be now possible, but hasn't given it a try ever since they worked on it. However, if were to sub module the entire*B*, that's the easiest way to go, but that will include other nonessential protos. They moved to another solution where they created a *git page* for *A* (containing the js files, which still lives under B), and wrote a js script in that will "get" the js files directly from an external project (in our case it will be js-btfs). They have examples of this on their Tronweb folder. 
What's your take on this ?

Reply:
@uchetron The git page solution sounds good

Reply:
@taiyangc great, just need pr from go-btfs-commons to be approved, in the interim will write a script here that will do the dependency retrieval.

Reply:
merged now, ready for the retrieval update, thx